velox/exec/Task.cpp

```diff
@@ -144,9 +144,24 @@ Task::Task(
       consumerSupplier_(std::move(consumerSupplier)),
       onError_(onError),
       pool_(queryCtx_->pool()->addScopedChild("task_root")),
-      bufferManager_(PartitionedOutputBufferManager::getInstance()) {}
+      bufferManager_(PartitionedOutputBufferManager::getInstance()) {
+  auto strategy = memory::MemoryManagerStrategy::instance();
+  if (strategy->canResize()) {
+    auto tracker = memory::MemoryUsageTracker::create(
```

Comment:
pool_ should have already been assigned tracker. Can we just set grow callback to it?